app/assets/images/2fa.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 6.87 11.12"><defs><style>.cls-1{fill:#5b616a;}.cls-2{fill:#fff;}</style></defs><title>2fa</title><g id="Layer_2" data-name="Layer 2"><g id="Layer_3" data-name="Layer 3"><path class="cls-1" d="M5.72,0H1.14A1.16,1.16,0,0,0,0,1.17V9.94a1.16,1.16,0,0,0,1.14,1.17H5.72A1.16,1.16,0,0,0,6.87,9.94V1.17A1.16,1.16,0,0,0,5.72,0Z"/><ellipse class="cls-2" cx="3.43" cy="9.79" rx="0.65" ry="0.66"/><rect class="cls-2" x="0.79" y="2.16" width="5.29" height="6.33"/></g></g></svg>
```

Comment:
In fact, looking at the PR you linked, it looks like any svg will have to be manually reexported.